analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -363,7 +363,7 @@ def add_arguments_to_parser(parser):
     output_opts.add_argument('-e', '--export',
                              dest="export",
                              required=False,
-                             choices=['html', 'json'],
+                             choices=['html', 'json', 'codeclimate'],
                              help="Specify extra output format type.")
```

Comment:
Please add a small description about the codeclimate format in the help message, something like it can be used for codeclimate and for gitlab integration and maybe a link to the specification would be nice too:
https://github.com/codeclimate/platform/blob/master/spec/analyzers/SPEC.md#data-types